### PR TITLE
Use is_boto3_error_code in 'standard' example

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

It looks like we didn't update some of our examples in the  AWS Guidelines to recommend is_boto3_error_code when working around a specific error is possible

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst

##### ADDITIONAL INFORMATION

CC: @jillr @s-hertel

### CODE REVIEW

File: docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst [hunk 1]
```diff
@@ -286,20 +286,18 @@ amounts of exception handling to existing modules, we recommend migrating the mo
 Note that it should normally be acceptable to catch all normal exceptions here, however if you
 expect anything other than botocore exceptions you should test everything works as expected.
 
-If you need to perform an action based on the error boto3 returned, use the error code.
+If you need to perform an action based on the error boto3 returned, use the error code and the
+``is_boto3_error_code()`` helper
```
Comment (s-hertel) [inline docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst]:
```suggestion
``is_boto3_error_code()`` helper.
```

File: docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst [hunk 2]
```diff
@@ -286,20 +286,18 @@ amounts of exception handling to existing modules, we recommend migrating the mo
...
        result = connection.describe_frooble(FroobleName=name)
-   except botocore.exceptions.ClientError as e:
-       if e.response['Error']['Code'] == 'FroobleNotFound':
-           workaround_failure()  # This is an error that we can work around
-       else:
-           module.fail_json_aws(e, msg="Couldn't obtain frooble %s" % name)
-   except botocore.exceptions.BotoCoreError as e:
+   except is_boto3_error_code('FroobleNotFound'):
+       workaround_failure()  # This is an error that we can work around
+   except (botocore.exceptions.BotoCoreError, botocore.exceptions.ClientError) as e:
```
Comment (s-hertel) [inline docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst]: You might want to add `# pylint: disable=duplicate-except` in case people copy-paste.

Comment (tremble) [reply docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst]:
```suggestion
   except (botocore.exceptions.BotoCoreError, botocore.exceptions.ClientError) as e:  # pylint: disable=duplicate-except
```

**Review by alinabuzachis — APPROVED** [review]
LGTM.

**Comment by Akasurde** [comment]
@tremble Thanks for the contribution. @s-hertel @alinabuzachis Thanks for the review.

**Pull request merged**